source-map.bs

```diff
@@ -274,6 +289,19 @@ fields represent mapped code that also has a mapped name.
 Note: Using file offsets was considered but rejected in favor of using line/column data to avoid becoming
 misaligned with the original due to platform-specific line endings. 
 
+### [=Mappings=] for generated JavaScript code
+
+Generated code positions that may have [=Mapping=] entries are defined in terms of *input elements*
+as per [=ECMAScript Lexical Grammar=]:
+
+1. [=Mapping=] entries must point to the first code point of the source text matched by
+    [=IdentifierName=], [=PrivateIdentifier=], [=Punctuator=], [=DivPunctuator=], [=RightBracePunctuator=],
+    [=NumericLiteral=] and [=RegularExpressionLiteral=].
+
+1. [=Mapping=] entries may point to any code point of the source text matched by
+    [=Comment=], [=HashbangComment=], [=StringLiteral=], [=Template=],
+    [=TemplateSubstitutionTail=], [=WhiteSpace=] and [=LineTerminator=].
```

Comment:
I agree with the contents, but I found the wording confusing (if mappings _must_ point to the first code point of things in the first item, how can they also point somewhere else)?

What about this?
```suggestion
Generated code positions that may have [=Mapping=] entries are defined in terms of *input elements*
as per [=ECMAScript Lexical Grammar=]. [=Mapping=] entries must point to either:

1. the first code point of the source text matched by
    [=IdentifierName=], [=PrivateIdentifier=], [=Punctuator=], [=DivPunctuator=], [=RightBracePunctuator=],
    [=NumericLiteral=] and [=RegularExpressionLiteral=].

1. any code point of the source text matched by
    [=Comment=], [=HashbangComment=], [=StringLiteral=], [=Template=],
    [=TemplateSubstitutionTail=], [=WhiteSpace=] and [=LineTerminator=].
```

Reply:
Thanks for the improved wording!